x-pack/test_serverless/shared/lib/security/types.ts

```diff
@@ -19,4 +19,7 @@ export enum ServerlessRoleName {
   ENDPOINT_OPERATIONS_ANALYST = 'endpoint_operations_analyst',
   ENDPOINT_POLICY_MANAGER = 'endpoint_policy_manager',
   READER = 'reader', // custom role to test lack of permissions
+  ADMIN = 'admin', // default Cloud role
+  SUPERUSER = 'system_indices_superuser', // this role is used to clean up the environment only and should
+  // not be used in any tests
```

Comment:
@tomsonpl @tomsonpl This should fix the failures. 

`admin` is Cloud default role, available both in serverless and ess deployments. It is a role that org owner got assigned and completely valid one to use in tests. But obviously the goal is to test features with minimal required role and `admin` is the last in the list.

`system_indices_superuser` is temporary solution while some Security solution teams didn't fix their tests, e.g. cloud security posture API tests depend on it. This role does not exist on MKI and tests using it might fail on MKI. We have a plan to remove it asap, but for now it has to stay to unblock testing in Kibana CI.